main.rb

```diff
@@ -1,20 +1,63 @@
-require_relative 'person'
-require_relative 'student'
-require_relative 'teacher'
-require_relative 'nameable'
+require_relative 'app'
+def display
+  {
+    1 => 'list_all_books',
+    2 => 'list_all_people',
+    3 => 'create_a_teacher',
+    4 => 'create_a_student',
+    5 => 'create_a_book',
+    6 => 'create_a_rental',
+    7 => 'list_rentals_for_a_person',
```

Comment:
- [ ] Please change this option to 'List all rentals for a given person id' according to the wireframe. And you need to ask for an id next, please change this method to be exactly like this: 
![image](https://github.com/vanina123/School-libary/assets/91486791/cab53911-3709-46ba-8b2b-c952d96e4ab1)
your app:
![image](https://github.com/vanina123/School-libary/assets/91486791/f2281963-4a20-4323-a8d2-97a137e60475)